source/code-snippets/usage-examples/findOneAndDelete.js

```diff
@@ -20,7 +25,7 @@ async function run() {
       // sort matched documents in ascending order by rating
       sort: { rating: 1 },
       // Include only the `title` and `fullplot` fields in the returned document
-      projection: { _id: 0, title: 1, fullplot: 1 },
```

Comment:
I believe we agreed to enable trailing commas (i know, i know), you should check your prettier config!